toplevel/ccompile.ml

```diff
@@ -108,8 +108,6 @@ let compile opts copts ~echo ~f_in ~f_out =
   in
   match copts.compilation_mode with
   | BuildVo ->
-      Flags.record_aux_file := true;
```

Comment:
Doesn't this change mean we never produce aux files?

Reply:
In theory the open aux file below should enable the generation for aux files, let me check again.

Reply:
Well, dune will complain if aux files are not generated so at least for the stdlib we are producing them.

Reply:
Looks like `Aux_file.recording ()` says `true` iff an output "method" for the aux file is set. I guess we need to carefully check if the old behavior is respected (eg the output method set when the flag used to be false).

The risk is generating them more often I guess